src/tests/Common/CLRTest.CrossGen.targets

```diff
@@ -287,18 +308,19 @@ if defined RunCrossGen2 (
     set "DOTNET_HeapVerify="
     set "DOTNET_ReadyToRun="
 
+    REM remove this?
```

Comment:
Nit: Let's use '#' instead of 'REM'. Let's avoid using Windows syntax on Unix to prevent getting the scripts from getting dirtier.

Reply:
@ivdiazsa this bit is actually a Windows code.

Reply:
Oh my bad. This file is just so confusing in general. Will close this comment then. Thanks for the clarification Jan!